src/main/java/racingcar/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,55 @@
+package racingcar.step2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    public static int splitAndSum(String text) {
+
+        if (isBlank(text)) {
+            return 0;
+        }
+        String[] tokens = tokenize(text);
+        return calculateSum(tokens);
+    }
+
+    //빈값 검증
+    private static boolean isBlank(String text) {
+        return text == null || text.isEmpty();
```

Comment:
```suggestion
        return text == null || text.isBlank();
```

공백 문자도 포함해서 체크할 수 있게 `isBlank()`를 사용하면 좋겠네요